### PR TITLE
Qa sprint4

### PR DESCRIPTION
**Pull request by jaffamonkey**
### Context

Updates to Sprint 4 and merged with the Sprint 3 branch - better to deal with this PR, rather than Sprint3-2 so closing that one

### Changes proposed in this pull request

Major updated to Sprint 4 cucumber tests, important update.

### Guidance to review

All changes are within the `features` folder only, asides from edits to README and added pa11ly.conf.js to `config` folder.

### CODE REVIEW

File: config/pa11y-ci.conf.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+var config = {
+    defaults: {
+      screenCapture: './_pa11y-screen-capture.png',
+      standard: 'WCAG2AAA',
+      level: 'error',
+      timeout: 5000,
+      wait: 1500
+    },
+    urls: [
+        '${HOST}',
```
Comment (tavy88) [inline config/pa11y-ci.conf.js]: Can we not read the HOST env here? That would avoid the horrible loop that comes after.

Comment (cwrw) [reply config/pa11y-ci.conf.js]: I was ignoring this and just wanted it to work till next time we picked up accessibility :P but I guess I could do it now, let me fix this file up

File: config/pa11y-ci.conf.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    defaults: {
+      screenCapture: './_pa11y-screen-capture.png',
+      standard: 'WCAG2AAA',
+      level: 'error',
+      timeout: 5000,
+      wait: 1500
+    },
+    urls: [
+      `${host}`,
+      `${host}/job-categories/government-services`,
```
Comment (smoothcontract) [inline config/pa11y-ci.conf.js]: Should be `/categories/government-services`

File: config/pa11y-ci.conf.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      `${host}/explore-occupations`,
+      `${host}/explore-occupations/results?utf8=✓&search=test`,
```
Comment (smoothcontract) [inline config/pa11y-ci.conf.js]: Don't think there's any need to include the utf8 query string param

File: config/pa11y-ci.conf.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      `${host}/check-your-skills`,
+      `${host}/check-your_skills/results?utf8=✓&search=manager`,
+      `${host}/find-training-courses`,
+      `${host}/next-steps`,
+      `${host}/explore-occupations/results?utf8=✓&search=construction`,
+      `${host}/find-training-course`
```
Comment (smoothcontract) [inline config/pa11y-ci.conf.js]: Should be `/find-training-courses`

File: config/pa11y-ci.conf.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    urls: [
+      `${host}`,
+      `${host}/job-categories/government-services`,
+      `${host}/explore-occupations`,
+      `${host}/explore-occupations/results?utf8=✓&search=test`,
+      `${host}/check-your-skills`,
+      `${host}/check-your_skills/results?utf8=✓&search=manager`,
+      `${host}/find-training-courses`,
+      `${host}/next-steps`,
+      `${host}/explore-occupations/results?utf8=✓&search=construction`,
```
Comment (smoothcontract) [inline config/pa11y-ci.conf.js]: Can remove utf8 query string param

File: features/qualifications_eligibility.feature [hunk 6]
```diff
@@ -1,11 +1,14 @@
 Feature: As a customer, I want *to confirm that I do not have qualifications at a level that mean the GHtR service is not best suited to me, So that I don't waste time using the service
```
Comment (smoothcontract) [inline features/qualifications_eligibility.feature]: Remove asterisk in title?

File: features/survey.feature [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Feature: User storyAs NRSI need to capture qualitative feedback from unmoderated users of the serviceSo that we can continually improve the service based on user feedback and insights
```
Comment (smoothcontract) [inline features/survey.feature]: Reword the start of this - not sure what NRSI is? But something more along the lines of `As a data analyst, I need...`

File: features/survey.feature [hunk 8]
```diff
@@ -1,4 +1,4 @@
-Feature: User storyAs NRSI need to capture qualitative feedback from unmoderated users of the serviceSo that we can continually improve the service based on user feedback and insights
+Feature: As a data analyst, I need need to capture qualitative feedback from unmoderated users of the service, so that we can continually improve the service based on user feedback and insights
```
Comment (smoothcontract) [inline features/survey.feature]: That's definitely needed then 😝

**Pull request merged**